delete.sh

```diff
@@ -0,0 +1,6 @@
+cd /data/kong && sudo rm -rf *
+cd /data/tomcat && sudo rm -rf *
+cd /data/rabbitmq && sudo rm -rf *
+cd /data/logs/kong && sudo rm -rf *
+cd /data/logs/rabbitmq && sudo rm -rf *
+cd /data/logs/tomcat && sudo rm -rf *
```

Comment:
maybe, we should add an option `--delete-data` to the `install` command to delete all the contents of the data directories. @pct960 any suggestions?